experimental/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { v4 as uuid } from "uuid";
+import { SharedMap } from "@fluid-experimental/fluid-framework";
+import { ContainerSchema } from "@fluid-experimental/fluid-static";
+import {
+    TinyliciousClient,
+    TinyliciousConnectionConfig,
+    TinyliciousContainerConfig,
+} from "..";
+
+describe("TinyliciousClient", () => {
+    before(() => {
+        const clientConfig: TinyliciousConnectionConfig = { port: 7070 };
+        TinyliciousClient.init(clientConfig);
+    });
+
+    let documentId: string;
+    beforeEach(() => {
+        documentId = uuid();
+    });
+
+    it("can create a container successfully", async () => {
+        const containerConfig: TinyliciousContainerConfig = { id: documentId };
+        const schema: ContainerSchema = {
+            name: documentId,
+            initialObjects: {
+                map1: SharedMap,
+            },
+        };
+        const container = await TinyliciousClient.createContainer(containerConfig, schema);
+
+        await new Promise<void>((resolve, reject) => {
+            container.on("connected", () => {
```

Comment:
Consider adding a reasonable timeout to reject the promise if "connected" event not raised

Reply:
I believe the testing framework has preset timeouts on each test. If it does I'm not sure if adding additional checks here is necessary.

Reply:
Yeah I believe it's 2s by default?  We have been more specific with timeouts in tests in some places, and it's definitely worth keeping in mind if these tests get more complicated (but it's pretty simple currently as is 🙂)

Reply:
What happens when "connected" fires before the listener is added? Is it possible?

Reply:
In this case it shouldn't be possible because we create new containers detached, which gives us the window to handle all that before attaching them.  I'm _pretty_ sure it's handled appropriately for the load case as well (and if it's not it's probably a bug and a failing test would be a good thing 🙃)

Reply:
Agreed. Thanks for clarifying.